tests/nnfw_api/src/GenModelTests/one_op_trains/Conv2D.test.cc

```diff
@@ -16,7 +16,9 @@
 
 #include "GenModelTrain.h"
 
-TEST_F(GenModelTrain, OneOp_Conv2D)
+namespace
+{
+CircleBuffers get_conv2d_test_model(const std::vector<int32_t> trainable_ops)
```

Comment:
(Optional) To avoid unnecessary copy.
```suggestion
CircleBuffers get_conv2d_test_model(const std::vector<int32_t> &trainable_ops)
```